clients/web/src/lib/components/atoms/form/Input/Input.svelte

```diff
@@ -0,0 +1,129 @@
+<script lang="ts" context="module">
+    export type InputSize = "sm" | "md" | "lg";
+</script>
+
+<script lang="ts">
+    import {slide} from "svelte/transition";
+    import shortid from "shortid";
+
+    import type {FormControlState} from "../form.types";
+
+    export let size: InputSize = "md";
+    export let label: string;
+    export let placeholder: string | undefined = undefined;
+    export let disabled: boolean = false;
+    export let state: FormControlState = "none";
+    export let error: string | undefined = undefined;
+
+    const labelId = `input_${shortid.generate()}`;
+    const errorId = `input-error_${shortid.generate()}`;
+</script>
+
+<div class="size-{size} state-{state}">
+    <label for={labelId}>{label}</label>
+
+    <div class="input-container">
+        {#if $$slots.addonLeft}
+            <div class="addon">
+                <slot name="addonLeft" />
+            </div>
+        {/if}
+
+        <input id={labelId} type="text" aria-describedby={error ? errorId : undefined} {placeholder} {disabled} />
+
+        {#if $$slots.addonRight}
+            <div class="addon">
+                <slot name="addonRight" />
+            </div>
+        {/if}
+    </div>
+
+    {#if error && state === "invalid"}
+        <span class="error" id={errorId} transition:slide>{error}</span>
+    {/if}
+</div>
+
+<style lang="postcss">
+    /* General styling */
+    label {
+        @apply block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300;
+    }
+
+    .input-container {
+        @apply flex items-stretch
+            rounded-lg border overflow-hidden
+            focus-within:ring-1 focus-within:ring-primary focus-within:border-primary
+            bg-gray-50 border-gray-300 text-gray-900 placeholder-gray-600
+            dark:bg-gray-700 dark:border-gray-600 dark:text-white dark:placeholder-gray-400;
```

Comment:
We are dark mode by default; without support for light mode, so we should swap this to remove the light mode values

Reply:
Why not support light mode? Its built in to all of the Flowbite components automatically

Reply:
The effort of supporting it both from a design & implementation perspective doesn't make sense (imo) compared to the potential for issues (i.e. a component written in one that doesn't have the responsive styles).

I would guess that the _overwhelming_ majority of our users are likely to use dark mode anyways.

Reply:
Fair, will remove

Reply:
I'd love to support light mode in the future though. Some people (particularly those with disabilities) strongly prefer light mode

Reply:
Fixed https://github.com/SprocketBot/sprocket/pull/333/commits/68996377c29f80fe013b265e8ac4d73998792f1d